src/conf_mode/load-balancing-inbound-ipvsadm.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env python3
+#
+# Copyright (C) 2020 VyOS maintainers and contributors
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 2 or later as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+
+from sys import exit
+
+from vyos.config import Config
+from vyos.configdict import dict_merge
+from vyos.template import render
+from vyos.util import cmd
+from vyos.util import dict_search
+from vyos.template import bracketize_ipv6
+from vyos.template import is_ipv6
+from vyos.template import is_ipv4
+from vyos.xml import defaults
+from vyos import ConfigError
+from vyos import airbag
+airbag.enable()
+
+lvs_config = '/tmp/ipvsadm.rules'
+
+def get_config(config=None):
+    if config:
+        conf = config
+    else:
+        conf = Config()
+    
+    base = ['load-balancing', 'inbound', 'ipvsadm']
+    conf = conf.get_config_dict(base, key_mangling=('-', '_'), get_first_key=True)
+    
+    if 'rule' in conf:
+        default_values = defaults(base + ['rule'])
+        for rule in conf['rule']:
+            conf['rule'][rule] = dict_merge(default_values, conf['rule'][rule])
+    
+    return conf
+
+def verify(config):
+    if not config:
+        return None
+    
+    if 'rule' in config:
```

Comment:
As the entire verification is done on the rules, why not spare one indention level and bail out early if rule is not in the config dict?

Reply:
done

Reply:
That is not what I meant. You should call return when it is not in the dict.